src/fctGraph/Utils/mutators/addPushOutFctForOutSlotWhichHasNone.js

```diff
@@ -0,0 +1,72 @@
+const { v4: uuidv4 } = require('uuid')
+
+const assertFCTGraphFirstArgument = require('../helpers/assertFCTGraphFirstArgument')
+const PushOut = require('../../../functionalities/PushOut')
+const InSlot = require('../../../slots/InSlot')
+const OutSlot = require('../../../slots/OutSlot')
+
+function assertIsOutSlot(slot) {
+  if (!(slot instanceof OutSlot)) {
+    throw new Error(`${slot} must be of type ${OutSlot.TYPE}`)
+  }
+}
+
+function assertSlotsFctExists(slot) {
+  const slotsParentFctExists = !!slot.functionality
+
+  if (!slotsParentFctExists) {
+    throw new Error(`${slot} parent fct can not be found by in the fctGraph`)
+  }
+}
+
+function assertDoesNotHaveConnectedOutputNode(slot) {
+  if (slot.isConnectedToOutputnode) {
+    throw new Error(`${slot} already has a connected OutputNode`)
+  }
+}
+
+function assertPrerequisites(fctGraph, outSlot) {
+  assertIsOutSlot(outSlot)
+  assertSlotsFctExists(outSlot)
+  assertDoesNotHaveConnectedOutputNode(outSlot)
+}
+
+function safeAddPushOutAndConnect(fctGraph, outSlot, pushOutFct) {
+  fctGraph.addFunctionality(pushOutFct)
+  const addConnectionResponse = outSlot.connectTo(pushOutFct.inSlots[0])
+
+  return addConnectionResponse
+}
+
+const DEFAULT_OPTS = {
+  pushOutData: {},
+}
+
+function addPushOutFctForOutSlotWhichHasNone(fctGraph, outSlot, opts = DEFAULT_OPTS) {
+  assertPrerequisites(fctGraph, outSlot)
+
+  const functionalityId = uuidv4()
+
+  const inSlotData = {
+    type: InSlot.TYPE,
+    dataType: outSlot.dataType,
+    name: 'Value In',
```

Comment:
can we make this just `value` ? `In` is redundant because the the type is already `InSlot.TYPE`

Reply:
updated!